src/lib/journey/_utilities/callback-mapper.svelte

```diff
@@ -71,7 +72,11 @@
   export let props:
     | Props
     | (Props & { recoveryCodes: Array<string> })
-    | (Props & { webAuthnValue: WebAuthnStepType });
+    | (Props & { webAuthnValue: WebAuthnStepType })
+    | (Props & {
+        journey: Maybe<StepOptions>;
+        recaptchaAction: string;
+      });
```

Comment:
I think we could probably leverage a discriminated union type but we would be making this type larger. However i think it would improve the intellisense through the switch case.

I'm going to pose this question to some groups and see if i'm on the right track